.gitignore

```diff
@@ -1 +1,4 @@
-.idea
\ No newline at end of file
+.idea
```

Comment:
Would you mind checking whether the generated project also gets these in the `.gitignore`, please?

Reply:
done